packages/bpk-component-datatable/src/BpkDataTable.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Table, AutoSizer } from 'react-virtualized';
+import { cssModules } from 'bpk-react-utils';
+import _ from 'lodash';
+
+import STYLES from './bpk-data-table.scss';
+import { BpkColumn } from './BpkColumn';
+
+const getClassName = cssModules(STYLES);
+
+const sortList = ({ sortBy, sortDirection, list }) => {
+  const sorted = _.sortBy(list, sortBy);
+  if (sortDirection === 'DESC') {
+    return _.reverse(sorted);
```

Comment:
No need for lodash: `sorted.reverse()`

Reply:
should also slice first, since reverse is (and `_.reverse`) are mutable operations: `sorted.slice().reverse()` :)